doc/source/whatsnew/v0.8.0.rst

```diff
@@ -156,16 +156,15 @@ Other new features
 New plotting methods
 ~~~~~~~~~~~~~~~~~~~~
 
-.. ipython:: python
-   :suppress:
+.. code-block:: python
 
    import pandas as pd
    fx = pd.read_pickle('data/fx_prices')
    import matplotlib.pyplot as plt
 
 ``Series.plot`` now supports a ``secondary_y`` option:
 
-.. ipython:: python
+.. code-block:: python
```

Comment:
Can you turn those again into ipython blocks now?

Reply:
@jbrockmendel do we need to revert this?

Reply:
this was reverted and then un-reverted, following https://github.com/pandas-dev/pandas/pull/28645#issuecomment-547322616

Reply:
cool